.env

```diff
@@ -113,5 +113,7 @@ WKHTMLTOPDF_PATH=/usr/local/bin/wkhtmltopdf
 WKHTMLTOIMAGE_PATH=/usr/local/bin/wkhtmltoimage
 ###< knplabs/knp-snappy-bundle ###
 
-###> ESRI API KEY ###
+###> ESRI Leaflet ###
 ESRI_API_KEY='yourkeyhere'
+#WORLD_VIEW_CODE='unitedStatesOfAmerica'
```

Comment:
Is it ok to leave it unset? I noticed you have it commented out.

Reply:
Yes, works fine.